### PR TITLE
[#1797] django-robots

### PR DESCRIPTION
**Pull request by mjstealey**
Removes any autogenerated robots.txt file from the static directory and instead uses the django-robots package to allow for custom defining the robots.txt directly from the admin panel in real-time.

- add django-robots to codebase allowing robots.txt configuration from admin panel
- updated pg.development.sql datbase to be current

### CODE REVIEW

**Comment by mjstealey** [comment]
Screenshot of admin panel
screen shot 2017-02-02 at 1 31 30 pm

Result in robots.txt
screen shot 2017-02-02 at 1 31 47 pm

File: hydroshare/settings.py [hunk 1]
```diff
@@ -172,10 +172,14 @@
 # a mode you'd pass directly to os.chmod.
 FILE_UPLOAD_PERMISSIONS = 0o644
 
-
 # Alternative tmp folder
 FILE_UPLOAD_TEMP_DIR = "/hs_tmp"
 
+# Sitemap for robots
+ROBOTS_SITEMAP_URLS = [
+    'http://localhost:8000/sitemap/',
```
Comment (pkdash) [inline hydroshare/settings.py]: @mjstealey  Is this URL setting right? Why it is not set to ```https```? Curious.

Comment (pkdash) [inline hydroshare/settings.py]: Looks like the above admin settings answers my question.

**Comment by mjstealey** [comment]
> @mjstealey Is this URL setting right? Why it is not set to https? Curious

This is going to vary from deployment to deployment. It's default setting is that of the development evironment. 
The settings.py file on production will reflect production appropriate values.

**Comment by pkdash** [comment]
@mjstealey I don't know much about robot.txt.  From me +1 though. If @hyi can also review it that would be better.

**Comment by hyi** [comment]
+1, I read up https://pypi.python.org/pypi/django-robots in order to review this PR and everything appears to be set up correctly. Also really nice to see your screen shots which helps me to see how django-robots works.

**Pull request merged**